prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```

Comment:
Hi Frank, `flex-basis` is applied to flex items (children) so you'll need to move this property to your rule set for the `.card-wrapper` selector.